templates/index.html

```diff
@@ -149,190 +370,211 @@ <h2>From our partners</h2>
           ) | safe
         }}
         <p class="p-pull-quote__quote">The combination of Ampere’s Arm-based servers with a provisioned virtualisation solution like Canonical’s Anbox Cloud delivers the flexible, high-performance and secure infrastructure that developers need in order to deliver a better user experience for consumers.</p>
-        <cite class="p-pull-quote__citation">Jeff Wittich<br>SVP of Products at <a href="https://amperecomputing.com/" class="p-link--external">Ampere</a></cite>
+        <cite class="p-pull-quote__citation">Jeff Wittich, SVP of Products at <a class="p-link--external" href="https://amperecomputing.com/">Ampere</a></cite>
```

Comment:
Any reason why we link Ampere Computing but not the others giving us a quote?

Reply:
@morphis Not sure I was just following the copy doc? I've made the other suggested changes, thanks. I'm going to remove the Deploy on AWS button for now as we don't have the link.

Reply:
Do you want to add links to the others giving us a quote?